pkg/apis/numaplane/v1alpha1/monovertexrollout_types.go

```diff
@@ -62,3 +62,8 @@ type MonoVertexRolloutList struct {
 func init() {
 	SchemeBuilder.Register(&MonoVertexRollout{}, &MonoVertexRolloutList{})
 }
+
+// IsHealthy indicates whether the MonoVertexRollout is healthy.
+func (mv *MonoVertexRolloutStatus) IsHealthy() bool {
+	return mv.Phase == PhaseDeployed
```

Comment:
When the phase is `pending`, it is progressing which should also be healthy.

Reply:
Updated